src/account.ts

```diff
@@ -414,11 +422,12 @@ export class Account {
         });
     }
 
-    private functionCallV2({ contractId, methodName, args = {}, gas = DEFAULT_FUNCTION_CALL_GAS, attachedDeposit, walletMeta, walletCallbackUrl }: FunctionCallOptions): Promise<FinalExecutionOutcome> {
+    private functionCallV2({ contractId, methodName, args = {}, gas = DEFAULT_FUNCTION_CALL_GAS, attachedDeposit, walletMeta, walletCallbackUrl, stringify }: FunctionCallOptions): Promise<FinalExecutionOutcome> {
         this.validateArgs(args);
+        const opts = stringify === undefined ? {} : {stringify};
```

Comment:
let's use a more specific name instead of `opts`

Reply:
is `options` specific enough? 

Feel free to suggest a new name.

Reply:
Removed this altogether.